tests/integration/study_data_blueprint/test_binding_constraints.py

```diff
@@ -784,17 +772,44 @@ def test_for_version_870(self, client: TestClient, user_access_token: str, study
 
         # Check that the matrices are daily/weekly matrices
         expected_matrix = np.zeros((366, 1))
-        for term_alias in ["lt", "gt", "eq"]:
-            res = client.get(
-                f"/v1/studies/{study_id}/raw",
-                params={
-                    "path": f"input/bindingconstraints/{bc_id_w_matrix}_{term_alias}",
-                    "depth": 1,
-                    "formatted": True,
-                },  # type: ignore
-            )
+        for operator in ["less", "equal", "greater", "both"]:
+            if operator != "both":
```

Comment:
You should use the `BindingConstraintOperator` class and use .value to get the string. The same goes for the rest of the tests